lms/djangoapps/instructor/views/api.py

```diff
@@ -2117,3 +2150,43 @@ def spoc_gradebook(request, course_id):
         'staff_access': True,
         'ordered_grades': sorted(course.grade_cutoffs.items(), key=lambda i: i[1], reverse=True),
     })
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_sales_admin
+@require_POST
+def make_invoice_transaction(request, course_id):  # pylint: disable=unused-argument
+    """
+    Adding invoice transaction  (payment or refund) for the Invoice.
+    invoice id should be valid for making transaction.
+    """
+    try:
+        invoice_id = request.POST.get('invoice_id', None)
```

Comment:
Style nitpick: the default for `get()` is `None`, so you don't need to explicitly specify it in the second argument.